tests/unit/iontests.js

```diff
@@ -142,88 +82,280 @@ define(['intern', 'intern!object', 'intern/dojo/node!fs', 'intern/dojo/node!path
+    'ion-tests/iontestdata/good/equivs/ints.ion',
+    'ion-tests/iontestdata/good/equivs/intsWithUnderscores.ion',
+    'ion-tests/iontestdata/good/equivs/listComments.ion',
+    'ion-tests/iontestdata/good/equivs/listsTrailingComma.ion',
+    'ion-tests/iontestdata/good/equivs/localSymbolTableAppend.ion',
+    'ion-tests/iontestdata/good/equivs/localSymbolTableNullSlots.ion',
+    'ion-tests/iontestdata/good/equivs/localSymbolTables.ion',
+    'ion-tests/iontestdata/good/equivs/nonIVMNoOps.ion',
+    'ion-tests/iontestdata/good/equivs/nopPadEmptyStruct.10n',
+    'ion-tests/iontestdata/good/equivs/nopPadNonEmptyStruct.10n',
+    'ion-tests/iontestdata/good/equivs/paddedInts.10n',
+    'ion-tests/iontestdata/good/equivs/sexpComments.ion',
+    'ion-tests/iontestdata/good/equivs/sexps.ion',
+    'ion-tests/iontestdata/good/equivs/structs.ion',
+    'ion-tests/iontestdata/good/equivs/symbols.ion',
+    'ion-tests/iontestdata/good/equivs/systemSymbols.ion',
+    'ion-tests/iontestdata/good/equivs/systemSymbolsAsAnnotations.ion',
+    'ion-tests/iontestdata/good/equivs/textNewlines.ion',
+    'ion-tests/iontestdata/good/equivs/timestampFractions.10n',
+    'ion-tests/iontestdata/good/equivs/timestampFractions.ion',
+    'ion-tests/iontestdata/good/equivs/timestampSuperfluousOffset.10n',
+    'ion-tests/iontestdata/good/equivs/timestamps.ion',
+    'ion-tests/iontestdata/good/equivs/timestampsLargeFractionalPrecision.ion',
+    'ion-tests/iontestdata/good/equivs/utf8/stringU0001D11E.ion',
+    'ion-tests/iontestdata/good/non-equivs/annotatedIvms.ion',
+    'ion-tests/iontestdata/good/non-equivs/annotations.ion',
+    'ion-tests/iontestdata/good/non-equivs/documents.ion',
+    'ion-tests/iontestdata/good/non-equivs/floats.ion',
+    'ion-tests/iontestdata/good/non-equivs/ints.ion',
+    'ion-tests/iontestdata/good/non-equivs/lists.ion',
+    'ion-tests/iontestdata/good/non-equivs/nulls.ion',
+    'ion-tests/iontestdata/good/non-equivs/sexps.ion',
+    'ion-tests/iontestdata/good/non-equivs/strings.ion',
+    'ion-tests/iontestdata/good/non-equivs/structs.ion',
+    'ion-tests/iontestdata/good/non-equivs/symbolTables.ion',
+    'ion-tests/iontestdata/good/non-equivs/symbolTablesUnknownText.ion',
+    'ion-tests/iontestdata/good/non-equivs/symbols.ion',
+    'ion-tests/iontestdata/good/non-equivs/timestamps.ion',
+    'ion-tests/iontestdata/good/timestamp/equivTimeline/leapDayRollover.ion',
+    'ion-tests/iontestdata/good/timestamp/equivTimeline/timestamps.ion',
+    'ion-tests/iontestdata/good/timestamp/leapDay.ion',
+    'ion-tests/iontestdata/good/timestamp/timestamp2011-02-20T19_30_59_100-08_00.10n',
+    'ion-tests/iontestdata/good/timestamp/timestamp2011-02.10n',
+    'ion-tests/iontestdata/good/timestamp/timestamp2011.10n',
+    'ion-tests/iontestdata/good/timestamp/timestampWithTerminatingEof.ion',
+    'ion-tests/iontestdata/good/timestamp/timestamps.ion',
+]);
+
+// /skiplists generated by toSkipList.pl
```

Comment:
Nit: stray `/`

Reply:
that `/` was intended to indicate an end marker (similar to `/` usage in html markup);  removed/clarified